### DISCUSSION TITLE
Конфиг для МТС Москва - помогите пожалуйста

### DISCUSSION THREAD

**Discussion by firestarter7924** [dovecoteescapee/ByeDPIAndroid#122] [Q&A]
По умолчанию не работает :( Я чайник. Попробовал потыкать, не проканало... Помогите плиз, кто разбирается. Заранее спасибо!

**Comment by nik-fray** [comment]
Вчера все заработало с этими настройками, а через пару часов - ошибка подключения, проверьте интернет
Приставка Сяоми тв бокс S 2gen
Андройд тв

**Reply by nik-fray** [reply]
> У ByeDPI есть такое на Андроид ТВ приставках. Просто зайдите в приложение нажмите Disconnect и Connect. Соединение переподключится и будет опять все работать. Наверно это не баг ByeDPI , потому-что установленные приложения VPN так же отключаются. Решить проблему "отваливания"ByeDPI после выключения тв помогает использование ByeDPI в режиме Proxy. Но для этого потребуется дополнительная настройка. Если часто "отваливается" ByeDPI в режиме VPN попробуйте установить более младшие релизы программы. Тут: https://github.com/dovecoteescapee/ByeDPIAndroid/releases

С любыми настройками вообще не подключается к ютьюбу... "Нет подключения к сети"
Вчера днем было, потом вылетело, теперь вообще не подключается...
С режимом прокси без изменения настроек тоже самое...

**Reply by lolyinseo** [reply]
Это просто надстройка над https://github.com/hufrea/byedpi 
Не используйте UI, а используйте "параметры командной строки" их можно взять из обсуждений byedpi по ссылке

я пока на 

`--proto http --auto=none --disorder 1  --auto=torst --tlsrec 1+s`

**Reply by Vesta-stack** [reply]
Да это лучше.

**Comment by lordot** [comment]
До вчера все работало с --disorder 1 --auto=torst --tlsrec 1+s --debug 1, но внезапно отвалилось на МТС. И я так понял у всех. Пробовал разный конфиг, пока не получается
